File: Codeforces/A - Boy or Girl.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
    string s;cin>>s;
    bool alpha[26]={false};
    int counter = 0;
    for(char i : s){
        if(!alpha[i - 'a']){
            alpha[i-'a']= true;
            counter++;
        }
    }
    if(counter%2!=0){
        cout <<  "IGNORE HIM!";
    }
    else cout << "CHAT WITH HER!";
}

File: Codeforces/A - Boy or Girl.java
import java.util.Arrays;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) {
        // write your code here
        Scanner scan = new Scanner(System.in);
        String s = scan.nextLine();
        int num=0;
        char[] c = s.toCharArray();
        Arrays.sort(c);
        for (int i=1;i<c.length;i++){
                if(c[i]==c[i-1]){
                    num++;
                }
            }
        num = c.length-num;
 
 
        if(num%2==0) {
            System.out.print("CHAT WITH HER!");
        }
        else {
            System.out.println("IGNORE HIM!");
        }
 
 
    }
}
